tests/test_pyth_client.py

```diff
@@ -16,14 +16,17 @@
 
 from pytest_mock import MockerFixture
 
-from mock import AsyncMock
+from mock import AsyncMock, Mock
 
 
 # Using constants instead of fixtures because:
 # 1) these values are not expected to be mutated
 # 2) these values are used in get_account_info_resp() and get_program_accounts_resp()
 #    and so if they are passed in as fixtures, the functions will complain for the args
 #    while mocking the respective functions
+V2_FIRST_MAPPING_ACCOUNT_KEY = 'BmA9Z6FjioHJPpjT39QazZyhDRUdZy2ezwx4GiDdE2u2'
+V2_PROGRAM_KEY = 'gSbePebfvPy7tRqimPoVecS2UsBvYv46ynrzWocc92s'
```

Comment:
declared as constants for the same reason as the below constants -- doesn't seems like there's a need to mock this?